src/ucp/core/ucp_request.inl

```diff
@@ -558,7 +558,7 @@ ucp_request_recv_data_unpack(ucp_request_t *req, const void *data,
         return status;
 
     default:
-        ucs_fatal("unexpected datatype=%lx", req->recv.datatype);
+        ucs_fatal("unexpected datatype=0x%" PRIx64, req->recv.datatype);
```

Comment:
remove space before PRIx64

Reply:
Same as the above reason. It seems that C++ requires a space between string literal and macro. IIUC this file is used by C++. correct?